fluss-filesystems/pom.xml

```diff
@@ -31,6 +31,7 @@
         <module>fluss-fs-hadoop-shaded</module>
         <module>fluss-fs-oss</module>
         <module>fluss-fs-s3</module>
+        <module>fluss-fs-gs</module>
```

Comment:
We will also need to copy `fluss-fs-gs` to `fluss/plugin` directory in `fluss-dist/assemblies/plugins.xml` just like other filesystems.